pkg/apis/scheduling/v1beta1/types.go

```diff
@@ -228,12 +237,14 @@ type SubGroupPolicySpec struct {
 
 	// SubGroupSize defines the number of pods in each sub-affinity group.
 	// Only when a subGroup of pods, with a size of "subGroupSize", can satisfy the network topology constraint then will the subGroup be scheduled.
+	// +kubebuilder:validation:Minimum=1
 	// +optional
 	SubGroupSize *int32 `json:"subGroupSize,omitempty" protobuf:"bytes,4,opt,name=subGroupSize"`
```

Comment:
kubebuilder:validation:Minimum=1 conflicts with optional, unless we have a default 1